sql/telemetry/feature_usage/query.sql

```diff
@@ -0,0 +1,114 @@
+SELECT
+  events_t.client_id AS clients_id,
+  events_t.submission_date AS submission_date,
+  COALESCE(protection_report, 0) AS protection_report,
+  COALESCE(pinned_tab, 0) AS pinned_tab,
+  COALESCE(default_browser, 0) AS default_browser,
+  COALESCE(many_uri, 0) AS many_uri,
+  COALESCE(pip, 0) AS pip,
+  COALESCE(filled_password, 0) AS filled_password,
+  COALESCE(remember_password, 0) AS remember_password,
+  COALESCE(sync_configured, 0) AS sync_configured
+FROM
+  (
+    SELECT
+      client_id,
+      submission_date,
+      IF(event_method = 'show' AND event_object = 'protection_report', 1, 0) AS protection_report,
+      IF(event_category = 'pictureinpicture' AND event_method = 'create', 1, 0) AS pip,
+    FROM
+      `moz-fx-data-shared-prod.telemetry.events`
+    WHERE
+      submission_date = @submission_date
+      AND normalized_channel = "release"
+  ) events_t
+FULL JOIN
```

Comment:
These joins may end up being untenable for the entire release population. Is there any chance we can use a 1% sample for this work, or at least start with a 1% sample?

Reply:
For my purposes, 1% is not possible.  Restricting to profiles that use particular features a particular number of days after first-seen-date makes numbers much smaller and the matching methods I use need fairly large sample sizes for statistical significance.